jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/test/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenterTest.java

```diff
@@ -504,4 +641,79 @@ public void testActiveSearchFiltersProcessDefinitionId(){
                                          any(Consumer.class));
     }
 
+    @Test
+    public void testIsAuthorizedForTaskAdminView() {
+        when(authorizationManager.authorize(any(ResourceRef.class),
+                                            eq(identity))).thenReturn(true,
+                                                                      false);
+
+        assertTrue(presenter.isUserAuthorizedForPerspective(TASKS_ADMIN));
+
+        final ArgumentCaptor<ResourceRef> captor = ArgumentCaptor.forClass(ResourceRef.class);
+        verify(authorizationManager).authorize(captor.capture(),
+                                               eq(identity));
+        assertEquals(TASKS_ADMIN,
+                     captor.getValue().getIdentifier());
+        assertEquals(ActivityResourceType.PERSPECTIVE,
+                     captor.getValue().getResourceType());
+
+        assertFalse(presenter.isUserAuthorizedForPerspective(TASKS_ADMIN));
+    }
```

Comment:
Could we add one more unit test for _openTaskView_ method or re-factor this one to also include arguments validation of _navigateToPerspective_ method?
In this way, I think branching would be properly tested and I don't need to add Selenium test which does that.